x-pack/legacy/plugins/apm/public/components/shared/TimestampTooltip/index.tsx

```diff
@@ -15,35 +15,84 @@ interface Props {
   precision?: 'days' | 'minutes' | 'seconds' | 'milliseconds';
 }
 
-function getPreciseTime(precision: Props['precision']) {
+function getPreciseTime(
+  precision: Props['precision'],
+  withSeparator: boolean = true
+) {
+  const separator = withSeparator ? ', ' : '';
   switch (precision) {
     case 'days':
       return '';
     case 'minutes':
-      return ', HH:mm';
+      return `${separator}HH:mm`;
     case 'seconds':
-      return ', HH:mm:ss';
+      return `${separator}HH:mm:ss`;
     default:
-      return ', HH:mm:ss.SSS';
+      return `${separator}HH:mm:ss.SSS`;
   }
 }
 
 function withLeadingPlus(value: number) {
   return value > 0 ? `+${value}` : value;
 }
 
-export function asAbsoluteTime({ time, precision = 'milliseconds' }: Props) {
-  const momentTime = moment(time);
+/**
+ * Returns the timezone set on momentTime.
+ * (UTC+offset) when offset if bigger than 0.
+ * (UTC-offset) when offset if lower than 0.
+ * @param momentTime Moment
+ */
+function formatTimezone(momentTime: moment.Moment) {
   const utcOffsetHours = momentTime.utcOffset() / 60;
   const utcOffsetFormatted = Number.isInteger(utcOffsetHours)
     ? withLeadingPlus(utcOffsetHours)
```

Comment:
Can you add a comment here (I should have done it in the first place)

```suggestion
    ? withLeadingPlus(utcOffsetHours) // custom timezone formatting
```

Reply:
I changed the name of the function and added a small description.